tests/__init__.py

```diff
@@ -132,6 +132,19 @@ def test_func(self):
                 except StaticValueException as e:
                     return_value = e.return_value
 
+                # Special-case; see https://github.com/hhursev/recipe-scrapers/issues/1372
+                if key == "description":
+                    return_value = return_value[:100]
+                    expect[key] = (
+                        expect[key].removesuffix("...")
+                        if len(expect[key]) > 100
+                        else expect[key]
+                    )
+                    self.assertTrue(
+                        len(expect[key]) <= 100,
```

Comment:
```suggestion
                    expect[key] = expect[key][:100].removesuffix("...")
                    self.assertLessEqual(
                        len(expect[key]),
                        100,
```

I think we can simplify here by trimming directly to 100 without checking if it’s already shorter, as it shouldn't change the output. Also using`assertLessEqual` for a slight readability increase.

Reply:
Using `assertLessEqual` sounds good to me.  I'm not so sure about the change to trim the suffix from already-less-than-100char strings though - that would make test cases for `description`s that genuinely end in `...` a bit unusual (some descriptions in the tests do already contain `...`, although at the moment those are greater than 100 chars in length).